variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "project_id" {
+  type        = string
+  description = "Id of GCP project."
+}
+
+variable "network_name" {
```

Comment:
We should name the network for the user instead of having a variable.
Note how this is done @ https://github.com/nullstone-modules/aws-network/blob/master/name.tf

Reply:
Added `name.tf` file as suggested and network name is being generated from that.

Reply:
Great!